hello_world.txt

```diff
@@ -0,0 +1 @@
+Hello, world!
```

Comment:
This is bad practice my dude

Reply:
If you don't like it you can neck yourself